crates/resolver/src/name_server/name_server_pool.rs

```diff
@@ -289,7 +289,6 @@ impl NameServerTransportState {
         self.0.insert(
             (ip, protocol),
             TransportState::Success {
-                connected_at: Instant::now(),
```

Comment:
this seems like a surprising change in a commit titled based on visibility tweaks.

Reply:
Expanding the commit message sounds good :+1: